frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.unit.spec.tsx

```diff
@@ -174,14 +175,28 @@ describe("Visualizations > PivotTable > PivotTable", () => {
 
     setup({ initialSettings: hiddenSettings });
 
-    rows.forEach(rowData => {
-      expect(screen.getByText(rowData[0].toString())).toBeInTheDocument();
+    expect(screen.getByText("Totals for bar1")).toBeInTheDocument();
 
-      [1, 2, 4, 5].forEach(colIndex => {
-        expect(
-          screen.queryByText(rowData[colIndex].toString()),
-        ).not.toBeInTheDocument();
-      });
+    expect(screen.getByRole("img", { name: /add/i })).toBeInTheDocument();
+
+    //Collapsed aggregates
+    [4, 5].forEach(colIndex => {
+      expect(
+        screen.queryByText(rows[0][colIndex].toString()),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByText(rows[1][colIndex].toString()),
+      ).not.toBeInTheDocument();
+    });
+
+    //non grouped values are still visible
+    [0, 1, 2, 4, 5].forEach(colIndex => {
+      expect(
+        screen.getByText(rows[2][colIndex].toString()),
+      ).toBeInTheDocument();
+      expect(
+        screen.getByText(rows[3][colIndex].toString()),
+      ).toBeInTheDocument();
```

Comment:
Can we do something about the code to avoid a need in a comment? Maybe extract row and column indexes into constants with some meaningful names?